test/xapi/track_test.rb

```diff
@@ -24,27 +24,11 @@ def test_default_track
     assert_equal slugs, track.slugs
 
     # default test pattern
-    files = [
-      "file.ext",
-      "test.ext",
-      "file_test.ext",
-      "TestFile.ext",
-      "spec.ext",
-      "sub/test/file.ext",
-      "sub/TestDir/file.ext",
-    ]
-    assert_equal %w(file.ext spec.ext), files.reject {|f|
-      f =~ track.test_pattern
-    }
+    assert_equal(/test/i, track.test_pattern)
```

Comment:
I get that testing the regex engine isn't useful, but neither is asserting on a hard coded value. The test as it was written was testing behavior—albeit behavior that was written in the test file (which isn't good); this tests an implementation.

I'd rather delete the test entirely, or switch it back to testing something in the actual code base.